examples/seismic/acoustic/operators.py

```diff
@@ -162,7 +163,7 @@ def GradientOperator(model, source, receiver, time_order=2, space_order=4, **kwa
     subs = dict([(t.spacing, dt)] + [(time.spacing, dt)] +
                 [(i.spacing, model.get_spacing()[j]) for i, j
                  in zip(v.indices[1:], range(len(model.shape)))])
-    return Operator([eqn] + [gradient_update] + receivers,
+    return Operator([eqn] + receivers + [gradient_update],
```

Comment:
That's what I meant, that's the order it should be, I must have messed it up debugging at some point and forget about it, sorry.